recipes/cgat-apps/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set version = "0.5.3" %}
+
+package:
+  name: cgat-apps
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/d7/05/6518cd6bf64bc0d15f474ea707979b8a47d502b2ee7c043d3a5e33f602c9/cgat-0.5.3.tar.gz
+  sha256: 2ea78bcee73c0aa2427466efe4dab3d6b035089bb42cc735480dafe5ceb2f191
+
+build:
+  skip: True # [not py3k]
+  number: 0
+  entry_points:
+    - cgat = cgat.cgat:main
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - python
+    - alignlib-lite
```

Comment:
Most of them don't need to be in the host section imho.

Reply:
Thanks, @bgruening for the feed-back, I have removed the unnecessary dependencies.